packages/ui/spacing/src/Spacing.tsx

```
@@ -0,0 +1,21 @@
+import { CSSProperties, forwardRef, HTMLAttributes } from "react";
+import { assignInlineVars } from "@vanilla-extract/dynamic";
+import { theme, dynamicSize } from "./computed.css";
+
+interface SpacingProps extends Omit<HTMLAttributes<HTMLDivElement>, "types"> {
+  size: number;
+  direction?: "vertical" | "horizontal";
+}
+
+const Spacing = forwardRef<HTMLDivElement, SpacingProps>(({ size, direction = "vertical", style, ...props }, ref) => {
+  const validatedSize = Math.max(0, isNaN(size) ? 0 : size);
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)